src/ontology/nbo-edit.owl

```diff
@@ -5419,13 +5421,15 @@ SubClassOf(obo:NBO_0000445 obo:NBO_0000001)
 
 # Class: obo:NBO_0000446 (visual pursuit)
 
-AnnotationAssertion(Annotation(oboInOwl:source "NBO:GVG") obo:IAO_0000115 obo:NBO_0000446 "Selectively track a moving object.")
+AnnotationAssertion(obo:IAO_0000115 obo:NBO_0000446 "Selective tracking that follows a moving object by eye.")
```

Comment:
Missing source

Reply:
This is just an intermediate solution to an old issue - we are not yet ready to reorganise the NBO, which will involve detailed academic sourcing.